src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/beanvalidation/BeanValidationQuickFix.java

```diff
@@ -150,6 +150,17 @@ private void removeStaticModifierCodeActions(Diagnostic diagnostic, JavaCodeActi
         final String name = Messages.getMessage("RemoveStaticModifier");
         codeActions.add(JDTUtils.createCodeAction(context, diagnostic, name, getParticipantId()));
     }
+
+    /**
+     * Returns simple name for the given fully qualified name.
+     */
+    protected static String getSimpleName(String fqName) {
+        int idx = fqName.lastIndexOf('.');
+        if (idx != -1 && idx != fqName.length() - 1) {
+            return fqName.substring(idx + 1);
+        }
+        return fqName;
+    }
```

Comment:
I recall seeing a method like this somewhere else in the project. Did you copy this from another class? We should be reusing code wherever possible instead of duplicating it.

Reply:
Found this in `RemoveParamAnnotationQuickFix.java`. I did not check whether this is the only other copy of such a method.

```
    protected static String getShortName(String qualifiedName) {
        final int i = qualifiedName.lastIndexOf('.');
        if (i != -1) {
            return qualifiedName.substring(i+1);
        }
        return qualifiedName;
    }
```

Reply:
Hi @mrglavas, I noticed that the `AbstractDiagnosticsCollector` class already has a `getSimpleName` method defined, so I updated the code to utilize `getSimpleName` from `AbstractDiagnosticsCollector` accordingly.

Reply:
Also Do we need to update `RemoveParamAnnotationQuickFix.java` as well to use the `getSimpleName` method from `AbstractDiagnosticsCollector`? I also noticed that some classes are still using the pattern `annotationName.substring(annotationName.lastIndexOf('.') + 1)` to get the short name. Should we refactor those classes to consistently use `getSimpleName` for retrieving short names?

Reply:
@anusreelakshmi934 You should open a new issue describing your suggested improvements to reduce code duplication. This PR addresses issue #870 and I believe what you currently have here is sufficient for that.

Reply:
Sure @mrglavas . I will open an Issue for the above suggestion to avoid code duplication. Thanks